src/index.tsx

```diff
@@ -659,31 +670,50 @@ export class Slider extends PureComponent<SliderProps, SliderState> {
         const interpolatedRawValues = this._getRawValues(
             interpolatedTrackValues,
         );
+        const minRawValue = Math.min(...interpolatedRawValues);
+        const minThumbValue = new Animated.Value(minRawValue);
+        const maxRawValue = Math.max(...interpolatedRawValues);
+        const maxThumbValue = new Animated.Value(maxRawValue);
+
+        const _value = values[0].__getValue();
+        const sliderWidthCoefficient =
+            containerSize.width /
+            (Math.abs(minimumValue) + Math.abs(maximumValue));
+        const startPositionOnTrack = _startFromZero
+            ? _value < 0 + step
+                ? (_value + Math.abs(minimumValue)) * sliderWidthCoefficient
+                : Math.abs(minimumValue) * sliderWidthCoefficient
+            : 0;
+
+        const minTrackWidth = _startFromZero
+            ? Math.abs(_value) * sliderWidthCoefficient - thumbSize.width / 2
```

Comment:
nit: I always prefer to do "*" instead of "/" (more efficent, not need to change hahahah 👍 ) `thumbSize.width * 0.5`